drivers/aufs/aufs.go

```diff
@@ -201,17 +231,23 @@ func (a *Driver) Exists(id string) bool {
 	return true
 }
 
+// AdditionalImageStores returns additional image stores supported by the driver
+func (a *Driver) AdditionalImageStores() []string {
+	var imageStores []string
+	return imageStores
+}
```

Comment:
This initializes `imageStores` to the zero value for a slice, which is `nil`, so the method could as easily return `nil` directly.

Reply:
Fixed